compose/service.py

```diff
@@ -234,7 +234,12 @@ def get_container(self, number=1):
         """Return a :class:`compose.container.Container` for this service. The
         container must be active, and match `number`.
         """
-        for container in self.containers(labels=['{0}={1}'.format(LABEL_CONTAINER_NUMBER, number)]):
+        from compose.project import OneOffFilter  # noqa
```

Comment:
This feels like an avoidable hack, can we move `OneOffFilter` into a `enums.py` (or equivalent)? Then we can import it from both `service.py` and `project.py` circumventing this import.